package.json

```diff
@@ -10,5 +10,8 @@
   },
   "devDependencies": {
     "husky": "^7.0.4"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1"
   }
```

Comment:
A to cudo też do backendu? Jeśli tylko front to do package.json w frontend

Reply:
done